rules/post_tsc_reminder_to_slack.yaml

```diff
@@ -0,0 +1,22 @@
+---
+name: "post_tsc_reminder_to_slack"
+pack: "st2community"
+description: "Post a reminder about the TSC meeting to Slack."
+enabled: true
+
+trigger:
+  type: "core.st2.CronTimer"
+  parameters:
+    timezone: "UTC"
+    day: 1st tue
```

Comment:
@nmaludy Just to make sure, did you double check if this kind of format works & recognized?

Reply:
I didn't check, no although it is documented and i looked at the code for the scheduler library.